otherlibs/stdune/fpath.ml

```diff
@@ -116,9 +116,7 @@ and rm_rf_dir path =
          deleted the directory. *)
       ())
 
-let rm_rf ?(allow_external = false) fn =
-  if (not allow_external) && not (Filename.is_relative fn) then
-    Code_error.raise "Path.rm_rf called on external dir" [ ("fn", String fn) ];
```

Comment:
This seems like a good check to keep when performing an operation this destructive.

Reply:
The check was redundant.

https://github.com/ocaml/dune/pull/5507/files#diff-dda6da045c3afdd7b140750abda581d73eb9dcfeaabf4711d1d3c330e5a57ffcL1051

Reply:
But yes in general I would prefer removing this function.